.github/workflows/core.yml

```diff
@@ -448,3 +448,29 @@ jobs:
           R -e "IRkernel::installspec()"
       - name: run tests
         run: ./mvnw verify -DskipRat -pl livy -am  -B
+
+  default-build:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Tune Runner VM
+        uses: ./.github/actions/tune-runner-vm
+      - name: Set up JDK 8
+        uses: actions/setup-java@v2
+        with:
+          distribution: 'temurin'
+          java-version: 8
+      - name: Cache local Maven repository
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.m2/repository
+            !~/.m2/repository/org/apache/zeppelin/
+            ~/.spark-dist
+            ~/.cache
+          key: ${{ runner.os }}-zeppelin-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-zeppelin-
+      - name: build without any profiles
+        run: ./mvnw clean package -DskipTests
```

Comment:
Please enter a new line at the end of the file :-)

Reply:
ok :) I'll be doing that